ci-operator/step-registry/openshift-qe/installer/bm/day2/metallb/openshift-qe-installer-bm-day2-metallb-commands.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+
+if [ ${BAREMETAL} == "true" ]; then
+  bastion="$(cat /bm/address)"
+  # Copy over the kubeconfig
+  sshpass -p "$(cat /bm/login)" ssh -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null root@$bastion "cat ~/bm/kubeconfig" > /tmp/kubeconfig
+  # Setup socks proxy
+  sshpass -p "$(cat /bm/login)" ssh -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null root@$bastion -fNT -D 12345
+  export KUBECONFIG=/tmp/kubeconfig
+  export https_proxy=socks5://localhost:12345
+  export http_proxy=socks5://localhost:12345
+  oc --kubeconfig=/tmp/kubeconfig config set-cluster bm --proxy-url=socks5://localhost:12345
+fi
+
+oc config view
+oc projects
+
+# Install the MetalLB operator
+if [ ${METALLB} == "true" ]; then
+  cat << EOF| oc apply -f -
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: metallb-system
+  labels:
+    openshift.io/cluster-monitoring: "true"
+EOF
+
+  cat << EOF| oc apply -f -
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: metallb-operator
+  namespace: metallb-system
+EOF
+
+  cat << EOF| oc apply -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: metallb-operator-sub
+  namespace: metallb-system
+spec:
+  channel: stable
+  name: metallb-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+EOF
+
+# Wait for the operator to be ready
+  until [ "$(kubectl get csv -n metallb-system | grep metallb-operator > /dev/null; echo $?)" == 0 ];
+    do echo "Waiting for MetalLB operator"
+    sleep 5
+  done
+  kubectl wait --for jsonpath='{.status.phase}'=Succeeded --timeout=10m -n metallb-system "$(kubectl get csv -n metallb-system -oname)"
```

Comment:
should we have the `10m` timeout be something we default to in the ref file, but allow the user to override the timeout?

Reply:
I guess most of the operators are installed way faster than 10 min. It is the same setup we have for the SRIOV one and so far has been consistent. Anyway, if you feel that the timeout should be configurable let me know and I'll quickly introduce the TIMEOUT var.

Reply:
ah ok - i am good with having or not having it. I didn't know if there was situations where we might want to bump that timeout.